bundles/org.eclipse.equinox.security.linux/META-INF/MANIFEST.MF

```diff
@@ -10,5 +10,4 @@ Bundle-Localization: fragment
 Eclipse-PlatformFilter: (osgi.os=linux)
 Export-Package: org.eclipse.equinox.internal.security.linux;x-internal:=true
 Automatic-Module-Name: org.eclipse.equinox.security.linux
-Eclipse-BundleShape: dir
```

Comment:
Adding this in https://github.com/eclipse-equinox/equinox/commit/fdebce601f328b46418d4c68100b772014c7723c,  was actually not necessary since this fragment does not contain any native code anymore.